front-end/src/data/KeyMeasures.js

```diff
@@ -8,62 +8,167 @@ function findSubMeasure(id) {
   }
 }
 
-const diagnosticsString = localStorage.getItem('diagnostics');
-let diagnostics = {};
-if (diagnosticsString) {
-  diagnostics = JSON.parse(diagnosticsString);
-} else {
-  diagnostics = {
+const defaultDiagnostics = {
+  "qualite-des-produits": {
+    "2019": {
+      valueBio: null,
+      valueFairTrade: null,
+      valueSustainable: null,
+      valueTotal: null,
+    },
+    "2020": {
+      valueBio: null,
+      valueFairTrade: null,
+      valueSustainable: null,
+      valueTotal: null,
+    }
+  },
+  "gaspillage-alimentaire": {
+    hasMadeWasteDiagnostic: false,
+    hasMadeWastePlan: false,
+    wasteActions: [],
+    hasDonationAgreement: false,
+  },
+  "diversification-des-menus": {
+    hasMadeDiversificationPlan: false,
+    vegetarianFrequency: null,
+    vegetarianMenuType: null,
+  },
+  "interdiction-du-plastique": {
+    cookingFoodContainersSubstituted: false,
+    serviceFoodContainersSubstituted: false,
+    waterBottlesSubstituted: false,
+    disposableUtensilsSubstituted: false,
+  },
+  "information-des-usagers": {
+    communicationSupports: [],
+    communicationSupportLink: null,
+    communicateOnFoodPlan: false,
+  },
+};
+
+const defaultFlatDiagnostic = {
```

Comment:
Il faudrait rajouter l'année ici

Reply:
Avec les changements de ce revue, le seule utilisation de ça sont les lignes que tu as noté en haut, qui seront supprimer dans la nouvelle PR. Cette PR aussi ajoute l'année pour les défauts. Donc ça va si je le laisse comme ça dans cette PR ?